modules/SofaGuiQt/src/sofa/gui/qt/RealGUI.cpp

```diff
@@ -1639,7 +1639,7 @@ void RealGUI::eventNewTime()
     {
         double time = root->getTime();
         char buf[100];
-        sprintf ( buf, "Time: %.3g s", time );
+        sprintf ( buf, "Time: %.3g,   Steps:  %o", time, frameCounter );
```

Comment:
```suggestion
        sprintf ( buf, "Time: %.3g,   Steps:  %i", time, frameCounter );
```
in fact it is wrong to use octal. steps 9 and 10 are missing in the count.